test/integration/test_events.py

```diff
@@ -6,22 +6,38 @@
 import json
 import os
 import shutil
+import stat
 
 from ansible_runner import run, run_async
 
 
-def test_basic_events(is_run_async=False,g_facts=False):
+@pytest.mark.serial
+@pytest.mark.parametrize('containerized', [True, False])
+def test_basic_events(containerized, container_runtime_available, is_run_async=False,g_facts=False):
+    if containerized and not container_runtime_available:
+        pytest.skip('container runtime(s) not available')
     tdir = tempfile.mkdtemp()
+    if containerized:
+        # container unable to access tempdir because of
+        # mkdtemp()'s minimal permissions
+        os.chmod(tdir, stat.S_IRWXU | stat.S_IRWXG | stat.S_IRWXO)
+
     inventory = "localhost ansible_connection=local"
     playbook = [{'hosts': 'all', 'gather_facts': g_facts, 'tasks': [{'debug': {'msg': "test"}}]}]
+    run_args = {'private_data_dir': tdir,
+                'inventory': inventory,
+                'playbook': playbook}
+    if containerized:
+        run_args.update({'process_isolation': True,
+                         'process_isolation_executable': 'podman',
+                         'container_image': 'ansible/ansible-runner',
+                         'container_volume_mounts': [f'{tdir}:{tdir}']})
+
     if not is_run_async:
-        r = run(private_data_dir=tdir,
-                inventory=inventory,
-                playbook=playbook)
+        r = run(**run_args)
```

Comment:
Using `**` vs. explicitly listing the key/value params will build up technical debt. We had to pay this dept in AWX when we added runner support to `tasks.py`